packages/pageflow-react/src/loadingSpinner/components/MediaLoadingSpinner.jsx

```diff
@@ -82,13 +82,17 @@ function preventScrollBouncing(e) {
   e.preventDefault();
 }
 
-function backgroundImageInlineStyles({firstPageBackgroundImageUrl, backgroundImage, blurStrength}) {
+function backgroundImageInlineStyles({firstPageBackgroundImageUrl, backgroundImage, blurStrength, backgroundImageX, backgroundImageY}) {
   const url = backgroundImage ? backgroundImage.urls.medium : firstPageBackgroundImageUrl;
   if (url) {
-    return {
+    var style = {
       backgroundImage: `url("${url}")`,
-      filter: 'blur('+blurStrength+'px)'
+      filter: 'blur('+blurStrength+'px)',
     };
+    if (backgroundImage) {
+      style.backgroundPosition = `${backgroundImageX}% ${backgroundImageY}%`
```

Comment:
This results in `undefined% undefined%` if no position has been set yet. For page backgrounds we [default to a centered position](https://github.com/codevise/pageflow/blob/943210b603ae94c83b110009ed63adbd7cc51d4a/packages/pageflow-react/src/components/BackgroundImage.jsx#L63).

Reply:
I tested this by using the image without setting fov values and then I didnot get undefined. can you please mention how can i reproduce undefined? Or I will try again

Reply:
This happens for every entry where the position of the custom loading spinner image has not been set yet. Example: https://staging-hosted-pageflow.codevise.de/admin/entries/132